dora/shell/src/main/java/alluxio/cli/fsadmin/report/JobServiceMetricsCommand.java

```diff
@@ -66,101 +56,14 @@ public JobServiceMetricsCommand(JobMasterClient JobMasterClient, PrintStream pri
    * @return 0 on success, 1 otherwise
    */
   public int run() throws IOException {
-    List<JobMasterStatus> allMasterStatus = mJobMasterClient.getAllMasterStatus();
-    String masterFormat = getMasterInfoFormat(allMasterStatus);
-    mPrintStream.printf(masterFormat, "Master Address", "State", "Start Time",
-        "Version", "Revision");
-    for (JobMasterStatus masterStatus : allMasterStatus) {
-      NetAddress address = masterStatus.getMasterAddress();
-      mPrintStream.printf(masterFormat,
-          address.getHost() + ":" + address.getRpcPort(),
-          masterStatus.getState(),
-          DATETIME_FORMAT.format(Instant.ofEpochMilli(masterStatus.getStartTime())),
-          masterStatus.getVersion().getVersion(),
-          masterStatus.getVersion().getRevision());
+    ObjectMapper objectMapper = new ObjectMapper();
+    JobServiceOutput jobServiceInfo = new JobServiceOutput(mJobMasterClient, mDateFormatPattern);
+    try {
+      String json = objectMapper.writeValueAsString(jobServiceInfo);
+      mPrintStream.println(json);
+    } catch (JsonProcessingException e) {
+      e.printStackTrace();
```

Comment:
before e.printStackTrace(), add a message and some outputs like
```
    try {
      String json = objectMapper.writeValueAsString(jobServiceInfo);
      mPrintStream.println(json);
    } catch (JsonProcessingException e) {
      mPrintStream.println("Failed to convert output to JSON. Check the command line log for the detailed error message.");
      LOG.error("Failed to output JSON object {}", jobServiceInfo);
```
Could you try if this works? Like will the jobServiceInfo be printed to the user in a human-readable form?

Reply:
In this place I can only import `org.jline.utils.Log` as the logger, is this package the same with others? The printed fail information is nice to have.

Reply:
create a new `LOG` variable at the beginning of the class like:
`private static final Logger LOG = LoggerFactory.getLogger(JobServiceMetricsCommand.class);`
where Logger and LoggerFactory are from `org.slf4j` imports